src/main/java/com/hackaton/gui/ResultWindow.java

```diff
@@ -44,7 +44,7 @@
              * Invoked when a window is in the process of being closed.
              * The close operation can be overridden at this point.
              *
-             * @param e
+             * @param e WindowEvent
```

Comment:
Poprawiasz komentarze wygenerowane przez IntelliJ? I nie trzymasz się konwencji
@param e - bla bla

Reply:
No zapomniałem o myślniku fakt

Reply:
Musi być myślnik pomiędzy typem a zmienną w Java?

Reply:
W ogóle Javadoc nie określa typu.

Reply:
Nie wcale, tylko IDE wyświetla Ci warning ;)

Reply:
Ty nie wiesz co to Javadoc? Nie Java tylko Javadoc? Jaki warning? W komentarzu?
Jednak poczytaj o Javadoc, bo nie wygląda jakbyś tego kiedyś używał.

Reply:
https://pl.m.wikipedia.org/wiki/Javadoc

Arek, masz rację z typami. Ale na wiki nigdzie nie pokazują myślników

Reply:
Bolek spór nie ma sensu, więc nie kłóćmy się o szczegóły. IDE wyświetla warninig, ale ok może coś źle zrobiłem. :)

Reply:
Gdzie ci wyświetla? Jak? Co zepsułeś? :P
A spór? Z tobą nie da się inaczej :D

Reply:
IDE pewnie wyświetla, bo jak zacząłeś pisać doc do funkcji, to ide chce, by umieścić w nim wszystkie parametry i return. Racja, akurat to czy jest myślnik czy go nie ma to mało istotne. (chociaż wiki mówi, że nie powinno być) ;-)

Reply:
Ale typu referencji też ;-)